### ISSUE TITLE
Unable to sign in with Google Account

### ISSUE THREAD

**Issue by CAM1977** [yuliskov/SmartTubeNext#62]
I have been unable to sign into SmartTube Next with my Google Account. I'm using a Nvidia Shield TV Pro 2019.  I go to YouTube.com/Activate, put in the on screen code and click through Google's screens until it says I'm hooked up to that version of YouTube. But on my TV nothing happens. If I click Done it just goes back to the main screen. Unlike other apps that immediately log me in. What am I doing wrong? I'm on the current 5.3 Beta. I'd love to try this app since I use your other Smart YouTube app. But not unless I can link it to my Google account.

**Comment by foreveryoungforty** [comment]
Try update 6.0 via Latest update-link https://github.com/yuliskov/SmartYouTubeTV/releases/download/beta/smarttube_beta.apk 
The automatic update message does not seem to work since 5.3

**Comment by CAM1977** [comment]
Thanks for the tip, updated to 6.0, but it still didn't log in on the
SmartTube Next app side, even though Google indicates that I'm signed in
from their side.  The app screen doesn't change after signing in (as it
always does with other apps, like Plex, HBO Now, etc). If I click Done it
just takes me back to the app main screen without any change.  It's getting
a bit frustrating.  Especially since I had no problem at all signing into
the Smart YouTube TV app.

**Comment by spwc321** [comment]
also having same issue/nvidia shield 2017 16gb

**Comment by rezpower** [comment]
I have the exact same problem and I don't have any addblock on my browser.
Do you mean disable adbloacker on my shield or on smarttub? can someone please explain?

**Comment by spwc321** [comment]
Check if you have any adblocker apps or dns changer apps installed on your
shield if so temporarily disable then try logging in again, if it still
fails then check accessibility settings and see if theres anything switched
on under services at the bottom of accessibility screen, temporarily
disable and try logging in again, i know the accessibility step sounds
random but i had some apps which depended on accessibility service to be
enabled and it stopped other apps from functioning correctly, two examples:

i have button mapper enabled under services and now bbc iplayer doesnt
work, i turn that off and bbc iplayer works again

I have anydesk installed and when i enable accessibility service for it the
"my apps" section of playstore fails to populate and remains blank.

To check accessibility settings go to

SETTINGS>DEVICE PREFERENCES>ACCESSIBILITY (NEAR BOTTOM OF MENU) Then check
the SERVICES part

Hope you get it working, also use your phone to go to YouTube/activate dont
do that part on the shield itself.

**Comment by kolslorr** [comment]
I have exact same issue with exact same setup. Don't have ad blockers too, but haven't checked under accessibility settings.

**Comment by rezpower** [comment]
Finally fixed with version beta 7.0
Thanks :)

**Comment by CAM1977** [comment]
Yes, finally works for me also now.  After dozens of times trying to log in with my account I was about to give up.  Thanks for continuing to work on what should be a great program, able to rival Smart YouTube, very soon.  Best of luck to you!

**Comment by blakegearin** [comment]
@CAM1977 Good to close?
